index.html

```diff
@@ -1,2 +1,3 @@
 <h1>Jenn is the best too so is Ramon, Jere, AJ, Joe</h1>
-<h2>Everyone needs to check out the podcast Sh!t You Don't Want to Talk About</h2>
\ No newline at end of file
+<h2>Everyone needs to check out the podcast Sh!t You Don't Want to Talk About</h2>
+<h3>Hello world! Please correct this tyop I made</h3>
```

Comment:
```suggestion
<h3>Hello world! Please correct this typo I made</h3>
```

Corrected typo from "tyop" to "typo"